faqs/galaxy/datasets_custom_genomes.md

```diff
@@ -0,0 +1,23 @@
+---
+title: What is a "Custom Reference Genome" ?
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Nurzhamalyrys]
+---
+
+- A reference genome contains the nucleotide sequence of the chromosomes, scaffolds, transcripts, or contigs for single species. It is representative of a specific genome build or release.
+
+- In Galaxy, a custom reference genome is a FASTA formatted dataset that can be used in place of a native reference genome with most tools.
+
+ - custom: a dataset from the history loaded by users
+ - native: local or cached by administrators 
+
+
+There are five basic steps to using a Custom Reference Genome:
```

Comment:
I think that this is where the actual FAQ starts, what do you think @Nurzhamalyrys?

Reply:
@beatrizserrano Hi, I think , yes, main part starts here. However, it is good for a reader to know  what  a custom genome is. How do you think?